quinn-proto/src/connection/mod.rs

```diff
@@ -699,7 +699,7 @@ impl Connection {
                         // Clamp the datagram to at most the minimum MTU to ensure that loss probes
                         // can get through and enable recovery even if the path MTU has shrank
                         // unexpectedly.
-                        usize::from(INITIAL_MTU)
+                        std::cmp::min(segment_size, usize::from(INITIAL_MTU))
```

Comment:
Nit: I like to spell this as `Ord::min(..)`, which is a little less noisy.

Reply:
```suggestion
                        segment_size.min(INITIAL_MTU as _)
```

Reply:
Personally I find the symmetric spelling easier to understand.

Reply:
Given it is a nit and there seems to be no consensus on the spelling I'm going to leave it.  Personally I have no preference and find all of the ways to write this fine.  I left the `usize::from()` part because that was how the code already did the cast.